assets/prometheus-k8s/prometheus.yaml

```diff
@@ -42,14 +42,16 @@ spec:
   - metrics-client-ca
   containers:
   - args:
+    - -request-logging=true
```

Comment:
Why request-logging is needed in this case?

Reply:
It's just for debugging :) I can't figure out why oauth-proxy isn't authorizing the requests I expect it to.